test/libsolidity/GasCosts.cpp

```diff
@@ -162,10 +162,10 @@ BOOST_AUTO_TEST_CASE(single_callvaluecheck)
 				a = b;
 			}
 			function f1(address b) public pure returns (uint c) {
-				return uint(b) + 2;
```

Comment:
This will cause the computation to be done in uint160 - not an issue in this test ,but it might cause bugs in contracts in general.